api/compression.md

```diff
@@ -11,14 +11,42 @@ policy](/compression/add_compression_policy/) for when to compress chunks.
 Advanced usage of compression allows users to [compress chunks
 manually](/compression/compress_chunk), instead of automatically as they age.
 
+Compressed chunks have the following limitations:
+- `ROW LEVEL SECURITY` is not supported.
+- Constraints are not fully supported:
+    - Unique constraints (`UNIQUE`) are restricted to compression `SEGMENTBY`
+      and `ORDER BY` columns.
+    - Primary Keys (`PRIMARY KEY`) are restricted to compression `SEGMENTBY`
+      and `ORDER BY` columns.
+    - Constraints are restricted to compression `SEGMENTBY` columns when
+    applicable. Otherwise they are unsupported.
+  
 ### Restrictions
 
-The current version does not support altering or inserting data into compressed
-chunks. The data can be queried without any modifications, however if you
-need to backfill or update data in a compressed chunk you will need to
-decompress the chunk(s) first.
+In general, compressing a hypertable does impose some limitations on the types
+of data modifications that can occur on data inside a compressed chunk. The
+table below provides a brief outline of allowed modifications based on the
+version of TimescaleDB you are currently running.
+
+|TimscaleDB Version|Supported data modifications on compressed chunks|
+|---|---|
+| 1.6 - 2.0 | Data and schema modifications supported. |
+| 2.1 - 2.2 | Schema may be modified on compressed hypertables. Data modification not supported. |
+| 2.3 | Schema modifications and basic insert of new data is allowed. Deleting, updating and some advanced insert statements are not supported |
 
 Starting with TimescaleDB 2.1, users have the ability to modify the schema
 of hypertables that have compressed chunks.
 Specifically, you can add columns to and rename existing columns of 
-such compressed hypertables.
\ No newline at end of file
+such compressed hypertables.
+
+Starting with TimescaleDB 2.3, users have the ability to insert data into
+compressed chunks, and enable compression policies on distributed hypertables.
```

Comment:
```suggestion
Starting with TimescaleDB 2.3, users have the ability to insert data into compressed chunks 
and to enable compression policies on distributed hypertables.
```

Reply:
I kind of disagree with this grammar suggestion. I think the oxford comma and parallel prepositions are used correctly here for technical documentation purposes.
Sources:
https://www.rd.com/article/oxford-comma-proper-use/
https://www.grammarly.com/blog/parallel-structure-prepositions/

Reply:
Scratch the parallel prepositions one. That needs fixing.